### PR TITLE
Deactivate

### PR DESCRIPTION
**Pull request by rbillings**
Deactivating the Treeherder-tests repo now that the code has been moved to https://github.com/mozilla/treeherder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,5 @@
 # Tests for Mozilla's Treeherder
-This repository contains tests for [Mozilla's Treeherder](https://treeherder.mozilla.org/).
+This repository holds automation code that was used for Mozilla's Treeherder website. All code has been moved to the [primary Treeherder repo](https://github.com/mozilla/treeherder).
```
Comment (stephendonner) [inline README.md]: Nit: s/holds/held

**Comment by stephendonner** [comment]
r+ after you resolve the conflicts; thanks!

Also, we should take care of following https://wiki.mozilla.org/QA/Execution/Web_Testing/Automation/Repositories#Deactivating_a_repository

I'd be happy to help with that, if you'd like, just let me know!

File: README.md (hunk 1 above)
Comment (rbillings) [inline README.md]: I would argue that it still holds code in the commit history & also matches previous deactivated repos such as https://github.com/mozilla/oneanddone.  I can be swayed if you feel strongly about it.

Comment (stephendonner) [inline README.md]: @rbillings nah, it's totally a nit.  Merge away after conflict resolution + final-green-button enablement!

**Review by stephendonner — APPROVED** [review]
r+

**Pull request merged**